src/Parser/PntsParser.js

```diff
@@ -105,14 +104,13 @@ function parseFeatureBinary(array, byteOffset, FTJSONLength) {
     if (parseJSON.BATCH_ID) {
         throw new Error('For pnts loader, BATCH_ID: not yet managed');
     }
-    // creation points with geometry and material
-    const points = new THREE.Points(geometry, material);
-    points.realPointCount = lengthFeature;
 
     // Add RTC feature
-    if (parseJSON.RTC_CENTER) {
-        points.position.fromArray(parseJSON.RTC_CENTER);
-    }
+    const offset = parseJSON.RTC_CENTER ?
+        new THREE.Vector3().fromArray(parseJSON.RTC_CENTER) : undefined;
 
-    return points;
+    return {
+        geometry,
+        offset,
+    };
```

Comment:
it seems there is no test on `PntsParser`

Reply:
> it seems there is no test on PntsParser

Yes.

Reply:
you could add it?

Reply:
No, because that's not the point of this PR.

Reply:
I added a test so we can hopefully move forward...